{"name": "test_openanaccount_a1_error_2", "status": "passed", "description": "账户开户失败的用例", "attachments": [{"name": "log", "source": "bc198858-ee8e-42ab-8fbb-9c330a4cf082-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "e817ccb3-1742-4b96-a029-7a4b74225fe9-attachment.txt", "type": "text/plain"}], "start": 1617349376349, "stop": 1617349378618, "uuid": "1351765d-c004-4e0b-b34b-c96f148ab5a5", "historyId": "93a67314a0fcb77fd4b65d6b74828418", "testCaseId": "11c4b2c12c3d6bddc23ee95d8dc02347", "fullName": "tests.test_a8_the_customer_account.TestTheCustomerAccount#test_openanaccount_a1_error_2", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_a8_the_customer_account"}, {"name": "subSuite", "value": "TestTheCustomerAccount"}, {"name": "host", "value": "DESKTOP-U13POE8"}, {"name": "thread", "value": "6480-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_a8_the_customer_account"}]}